modules/developers-guide/examples/candid-ui.adoc

```diff
@@ -0,0 +1,28 @@
+== Testing functions using a web interface
+
+The canister interface description language—often referred to as Candid or more generally as the IDL—provides a common syntax for specifying input arguments and return values.
+The language performs automatic type mapping so you can use the same syntax to interact with the `+dfx+` command-line interface, JavaScript, {proglang}, and other supported languages or tools.
+
+The language can also automatically generate a default web interface that exposes your program's functions for testing and debugging.
```

Comment:
```suggestion
Based on the type signature of the actor, Candid provides a web interface that allows you to call canister functions in a graphical manner.
```

Reply:
web interface and graphical manner sort of say the same thing, so I restored the reason for using the graphical user interface--testing and debugging.